src/PaintChannelPresets.cpp

```
@@ -0,0 +1,136 @@
+    pc->setBrushStrength(p.strength);
+    pc->setBrushFalloff(p.falloff);
+    pc->setBrushColor(QColor(p.colorR, p.colorG, p.colorB));
+    // Stamp vs round brush.
+    if (!p.stamp.isEmpty()) {
+        pc->setFootprintType(static_cast<int>(BrushFootprint::FootprintType::StampImage));
+        pc->setActiveStampName(p.stamp);
+    } else {
+        pc->setFootprintType(static_cast<int>(BrushFootprint::FootprintType::Round));
+    }
+
+    emit presetApplied(name);
```

Cppcheck (2.21.0)

[error] There is an unknown macro here somewhere. Configuration is required. If Q_ENUM is a macro then please configure it.

(unknownMacro)